src/addition.jl

```diff
@@ -1,36 +1,36 @@
 
 # between pdmat and pdmat
 
-+(a::PDMat, b::AbstractPDMat) = PDMat(a.mat + full(b))
-+(a::PDiagMat, b::AbstractPDMat) = PDMat(_adddiag!(full(b), a.diag))
-+(a::ScalMat, b::AbstractPDMat) = PDMat(_adddiag!(full(b), a.value))
-if isdefined(Base.SparseArrays, :CHOLMOD)
++(a::PDMat, b::AbstractPDMat) = PDMat(a.mat + Matrix(b))
++(a::PDiagMat, b::AbstractPDMat) = PDMat(_adddiag!(Matrix(b), a.diag))
++(a::ScalMat, b::AbstractPDMat) = PDMat(_adddiag!(Matrix(b), a.value))
+if isdefined(SparseArrays, :CHOLMOD)
```

Comment:
This will actually always be `false`, since the sparse linear algebra functionality was moved to the stdlib package SuiteSparse rather than SparseArrays. You could do
```julia
const HAVE_CHOLMOD = isdefined(SuiteSparse, :CHOLMOD)

if HAVE_CHOLMOD
    # methods and whatnot
end
```